source/connect/connections.txt

```diff
@@ -0,0 +1,73 @@
+.. _connections-sidebar:
+
+===================
+Connections Sidebar
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The :guilabel:`Connections Sidebar` shows MongoDB deployments that
+you can interact with through MongoDB Compass. 
+
+.. note::
+
+   .. include:: /includes/multiple-connections.rst
+
+
+The :guilabel:`Connections` sidebar on the left of the Compass window
+contains the following options:
+
+- :icon-lg:`Plus` icon to add new connections
+- :icon-fa5:`ellipsis-h` icon to import and export data
+- Search bar to search connections
+- guilabel:`Connections` list
+
+Connections List
+----------------
+
+The :guilabel:`Connections` list in the sidebar contains your
+:ref:`favorite <favorite-connections>` and saved connections. Favorite
+connections have a :icon-fa5:`star` next to the connection name.  The
+:guilabel:`Connections` list displays favorite connections in alphabetical
+order first. 
+
+Connect to MongoDB
+------------------
+
+To connect to a saved or favorite deployment, click the name of the
+connection in the :guilabel:`Connections` list. Once you connect,
+Compass shows a green dot next to the connection name. If a connection
+error occurs, Compass shows a red triangle next to the connection name. 
```

Comment:
We also show toasts. If there's an error the error toast will stay until the user dismisses it. Iti contains a REVIEW button to see the error in the connect form modal.

Reply:
I added a information about toasts for connections and errors.